Source/Krypton Toolkit/Krypton.Toolkit.Suite.Extended.Data.Visualisation/Scott Plot/Plottable/BarPlot.cs

```diff
@@ -101,10 +109,12 @@ private void RenderBarVertical(PlotDimensions dims, Graphics gfx, double positio
             }
 
             if (ShowValuesAboveBars)
+            {
                 using (var valueTextFont = GDI.Font(Font))
```

Comment:
braces placement in using can be simplified (ReSharper will do that)

Reply:
Done